docs/pages/Tests/OIOProvider/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+
+export default class Tests extends Component {
```

Comment:
Class name is a little generic and is the same as the class name in `docs/pages/Tests/index.js`. Should maybe be `OIOProviderTest`?